### PR TITLE
Issue #15218: Formatted input file for section 4.8.5.4 Field Annotations to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java [hunk 1]
```diff
@@ -4,66 +4,51 @@
...
+  float pi = 3.14f;
 
-    @SomeAnnotation1 @SomeAnnotation3(x = 14)
-    /** // violation 'Javadoc comment is placed in the wrong location.'
-     * testing.
-     */
-    List<String> list = new ArrayList<>();
+  @SomeAnnotation1 @SomeAnnotation3(x = 14)
+  /** Testing. */ // violation 'Javadoc comment is placed in the wrong location.'
+  List<String> list = new ArrayList<>();
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:37]: Formatter modified this code to something like:

```java
  @SomeAnnotation1
  @SomeAnnotation3(x = 14)
  /** // violation 'Javadoc comment is placed in the wrong location.' testing. */
  List<String> list = new ArrayList<>();
```

but this way we were not able to detect the violation message so I modified it a little bit

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java [hunk 2]
```diff
@@ -4,66 +4,51 @@
...
-    @SuppressWarnings("all")
-    @SomeAnnotation1
-    /** // violation 'Javadoc comment is placed in the wrong location.'
-     * testing.
-     */
-    InputFieldAnnotations obj = new InputFieldAnnotations();
+  @SuppressWarnings("all")
+  @SomeAnnotation1
+  /** Testing. */ // violation 'Javadoc comment is placed in the wrong location.'
+  InputFieldAnnotations obj = new InputFieldAnnotations();
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:42]: modified the violation message placement, reason mentioned above

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java [hunk 3]
```diff
@@ -4,66 +4,51 @@
...
+  public @interface SomeAnnotation3 {
+    int x();
+  }
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1
-    @SomeAnnotation2
-    String name = "Zops";
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:20]: please restore all annotation related formatting.
https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations

>Annotations applying to a field also appear immediately after the documentation block, but in this case, multiple annotations (possibly parameterized) may be listed on the same line;

may be, but does not have to.
User is ok to do formatting like this , if user use formatter and it make it single line, ok it is his choice, we just do not need to make violation on what formatter is doing, as both approaches are valid.

**Review by romani — CHANGES_REQUESTED** [review]
item

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java (hunk 3 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:20]: done

**Review by romani — APPROVED** [review]
ok to merge.

formatting on unrelated is good.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**